cost-analyzer/templates/kubecost-agent-secret-template.yaml

```diff
@@ -1,11 +1,13 @@
 {{- if .Values.agent }}
+{{- if or .Values.createAgentSecret (not (hasKey .Values "createAgentSecret")) }}
```

Comment:
Suggestion... just replace lines 1 and 2 with:
```
{{- if .Values.agentKey }}
```

That way if I don't set the key, it isn't created..

Reply:
Perfect. Thanks for the suggestion! Revised it.